spec/ransack/helpers/form_helper_spec.rb

```diff
@@ -399,6 +399,34 @@ module Helpers
         it { should match /example_name_eq/ }
       end
 
+      describe '#search_form_for with config set to remove sort order indicators' do
+        before do
+          Ransack.configure { |c| c.hide_sort_order_indicators = true }
+        end
+        subject { @controller.view_context
+          .sort_link(
+            [:main_app, Person.search(sorts: ['name desc'])],
+            :name,
+            controller: 'people'
+          )
+        }
+        it { should match /Full Name/ }
```

Comment:
If all this is confusing, don't worry about it and let me know, and I'll just commit it and clean it up.